test/membrane_live_web/controllers/webinar_controller_test.exs

```diff
@@ -79,6 +74,30 @@ defmodule MembraneLiveWeb.WebinarControllerTest do
     end
   end
 
+  describe "show" do
+    setup [:create_webinar]
+
+    test "valid request", %{
+      conn: conn,
+      webinar: %{uuid: webinar_uuid} = webinar
+    } do
+      conn = get(conn, Routes.webinar_path(conn, :show, webinar))
+
+      assert %{
+               "uuid" => ^webinar_uuid,
+               "description" => "some description",
+               "presenters" => [],
+               "start_date" => "2022-07-17T10:20:00",
+               "title" => "some title"
+             } = json_response(conn, 200)["webinar"]
+    end
+
+    test "reject if user is not authorized", %{conn: conn, webinar: webinar} = context do
+      callback = &get(&1, Routes.webinar_path(conn, :show, webinar))
```

Comment:
I would move `Routes.webinar_path(conn, :show, webinar)` to variable or use `then` and pipe it.

Reply:
What about this?